test/ext/base_lint.py

```diff
@@ -192,6 +207,9 @@ class CaseUnrelated(DoesNotIncludeUnexpectedFocused):
             a, *b, c = range(3)
             a, b, c = range(3)
             [a, b, c] = range(3)
+            
+            # Chained assignment should not cause errors
+            a = b = 3
```

Comment:
Since the assignment parsing bits must inspect _every_ chained assignment, I might include other interesting forms of them here, too
```suggestion
            a = b = 3
            a = b, c = (0, 1)
            a = b, *c = (0, 1)
            a = [b, c] = (0, 1)
```